src/shared/dissect-image.c

```diff
@@ -413,9 +414,11 @@ int dissect_image(
                         if (r < 0)
                                 return r;
 
+                        m->verity = !!root_hash;
```

Comment:
so far, the "verity" boolean so far was set when both a root hash hash was set and verity data was found in the image. And "can_verity" was set if verity data was found but no root hash specified.

I figure dissect_image() should also have a verity_data argument that it uses for initializing can_verity.

i.e.

```c
m->verity = verity_data && root_hash;
m->can_verity = !!verity_data;
```

Reply:
Ok, done